generator/src/main/resources/endpointDefinitions/DatabaseConnector.yaml

```diff
@@ -36,6 +36,8 @@ memberVariables:
     type: String
   - name: useKerberosAuth
     type: Boolean
+  - name: jdbcDriverId
+    type: Integer
```

Comment:
I should have caught this before, but we need to store the name of the jdbc driver here as well, not just the id. Then during setup, the name, not the id needs to be used. This is very similar to the file format and file metadata object relationship.

There is no guarantee from engine to engine that the jdbc driver will have the same id, so similar to the reason that we needed to use the name in the zip file, we need to use the name here as well, otherwise, on restore we may end up assigning the wrong jdbc driver to the connector.

There exists a map at `com.delphix.masking.initializer.maskingApi.SetupDriver#fileFormats` that is used to go from file format name -> file format id on the target engine. You will probably need to do something similar.